src/Routes.tsx

```diff
@@ -116,7 +116,7 @@ const CommonRoutesFeatures = ({ userModel }: CommonRoutesFeaturesProps) => {
       currentApplicationAtom.init.dispatch();
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  }, [pathname]);
```

Comment:
why not add all dependencies?

Reply:
Because I wanted to call it only once the app loaded.
To have hook like coomponentDidMount we need [] as dependencies, and there is no way without // eslint-disable-next-line  to make it pass

Reply:
from my experience it's ok to skip react-hooks/exhaustive-deps if you really need it